Packs/Okta/ReleaseNotes/2_2_6.md

```diff
@@ -0,0 +1,12 @@
+
+#### Integrations
+##### Okta IAM
+- The **iam-get-user** command will now try to retrieve user details by ID, then by username, then by email. 
+
+
+#### Mappers
+##### User Profile - Okta (Incoming)
+- Cortex XSOAR `Username` field in Cortex XSOAR will now be mapped from ServiceNow field `login`, instead of `userName` field.
+
+##### User Profile - Okta (Outgoing)
+- **Breaking Change:** Changed mapping of `login` field from ServiceNow to `Username` field in Cortex XSOAR instead of `Email`.
```

Comment:
```suggestion
- Cortex XSOAR `Username` field in Cortex XSOAR will now be mapped from Okta field `login`, instead of `userName` field.

##### User Profile - Okta (Outgoing)
- **Breaking Change:** Changed mapping of `login` field from Okta to `Username` field in Cortex XSOAR instead of `Email`.
```